config/groups.yaml

```
@@ -276,3 +276,9 @@
     - "2591194"
     - "2566752"
     - "2565473"
+- name: "Taylorians"
+members:
+    -"2885593"
+    -"2819226"
+    -"2829182"
+    -"2806845"
```

This is invalid YAML, the indentation and spacing matters:
```suggestion
- name: "Taylorians"
  members:
    - "2885593"
    - "2819226"
    - "2829182"
    - "2806845"
```